lms/djangoapps/django_comment_client/utils.py

```diff
@@ -652,7 +659,7 @@ def add_courseware_context(content_list, course, user, id_map=None):
             content.update({"courseware_url": url, "courseware_title": title})
 
 
-def prepare_content(content, course_key, is_staff=False, course_is_cohorted=None):
+def prepare_content(content, course_key, is_staff=False, discussion_division_enabled=None):
```

Comment:
discussion_division_enabled indicates a boolean to me it should be defaulted to True or False, not None.